be/src/exprs/hybrid_set.h

```diff
@@ -668,7 +668,7 @@ class StringValueSet : public HybridSetBase {
                      const doris::vectorized::NullMap* null_map,
                      doris::vectorized::ColumnUInt8::Container& results) {
         const auto& col = assert_cast<const doris::vectorized::ColumnString&>(column);
-        const uint32_t* __restrict offset = col.get_offsets().data();
+        const auto& offset = col.get_offsets();
```

Comment:
1. why this modificial works?
2. may be we can apply the change to other places like L672